src/app/patient-summary/components/resources/Patient/Patient.tsx

```diff
@@ -3,8 +3,8 @@ import InfoRow from '../../InfoRow';
 import ConnectionDetails from './ConnectionDetails';
 import PatientIdentifiers from './PatientIdentifiers';
 import { StyledSectionTypography } from '@/app/components/typography/StyledTypography';
-import { Patient as PatientType } from '@/app/patient-summary/types/patient.types';
-export default function Patient({ data }: { data: PatientType[] }) {
+import { IResourceType } from '@/app/patient-summary/types/fhir.types';
+export default function Patient({ data }: { data: IResourceType['Patient'] }) {
```

Comment:
Hello @yassinedorbozgithub,

Currently, I'm handling the Patient component as if it receives an array of patients, even though we're certain that there can't be more than one Patient per bundle. I took this approach because Patient appears to be the only resource that shouldn't have duplicates in a bundle, so I'm accessing it using data[0] inside the Patient component.

I haven't adjusted the logic to check whether we're receiving one or more entries for other resource types, as this seemed to be a special case for Patient.

However, if you think it makes sense to refactor this approach, I'm open to making changes.

Reply:
Hi @amalessid, 100% i will refactor the code according to the new changes.
Thank you for the feedback :pray: